modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -682,8 +682,9 @@ public class ApiClient {
 
     Entity<?> entity = serialize(body, formParams, contentType);
 
-    Response response;
+    Response response = null;
```

Comment:
@wing328 Local variables are slightly different; the compiler never assigns a default value to an uninitialized local variable. If I do not initialize the response variable, in compilation I get error "variable response might not have been initialized"